net/lp_rpc.proto

```diff
@@ -154,3 +157,56 @@ message NotifySegment {
     int64 taskId   = 16;
     bytes profiles = 17;
 }
+
+// Required parameters for probabilistic micropayment tickets
+message TicketParams {
```

Comment:
I think @j0sh's note about adding the recipient's ETH address in here makes sense.

@yondonfu wdyt?

Reply:
Yeah makes sense to me. Might be worth updating the Go [TicketParams](https://github.com/livepeer/go-livepeer/blob/master/pm/ticket.go#L19) type to include the recipient's ETH address as well for consistency, but we can address that separately from this PR.